package.json

```diff
@@ -31,8 +28,8 @@
     "tslib": "2.7.0"
   },
   "devDependencies": {
+    "@types/react": "^18.3.11",
```

Comment:
**Suggestion:** Ensure that the version of `@types/react` matches the version of `react` to prevent potential type mismatches and compatibility issues. [best practice, importance: 8]
```suggestion
    "@types/react": "^19.0.0",
```